<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CoqEditFilters</class>
 <widget class="QWidget" name="CoqEditFilters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,0">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTableWidget" name="table_filters">
     <property name="showGrid">
      <bool>false</bool>
     </property>
     <property name="columnCount">
      <number>1</number>
     </property>
     <attribute name="horizontalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <column/>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0,0">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1,0">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Column:</string>
         </property>
         <property name="buddy">
          <cstring>list_columns</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="list_columns">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="check_before_transformation">
         <property name="text">
          <string>&amp;Apply before transformation</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>&amp;Relation:</string>
         </property>
         <property name="buddy">
          <cstring>radio_OP_EQ</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_OP_EQ">
         <property name="text">
          <string>equals</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_OP_NE">
         <property name="text">
          <string>does not equal</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_OP_LT">
         <property name="text">
          <string>is less than</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_OP_LE">
         <property name="text">
          <string>is less or equal to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_OP_GT">
         <property name="text">
          <string>is greater than</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_OP_GE">
         <property name="text">
          <string>is greater or equal to</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_OP_MATCH">
         <property name="text">
          <string>matches</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio_OP_NMATCH">
         <property name="text">
          <string>does not match</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,1,0,0,1">
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&amp;Value:</string>
         </property>
         <property name="buddy">
          <cstring>edit_value</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QLineEdit" name="edit_value"/>
       </item>
       <item row="2" column="0" colspan="3">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>195</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="button_add">
         <property name="text">
          <string>&amp;Add</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPushButton" name="button_remove">
         <property name="text">
          <string>&amp;Remove</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>195</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0" rowspan="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="2" rowspan="2">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
